.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,11 +1,11 @@
 <!--- Provide a general summary of your changes in the Title above -->
 
-## Checklist for submitting an `awesome` plugin, theme, or resource:
-<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
-<!--- More detailed instructions are in the CONTRIBUTING.md document - please read it if you have a question!-->
+## Checklist for submitting a resource to `awesome-hyper`:
+<!--- Go over all the following points, and put an `x` in all the boxes that have been completed. -->
+<!--- More detailed instructions are in the project's CONTRIBUTING.md - please read it if you have a question!-->
 <!--- If you're still unsure about any of these, don't hesitate to ask. We're here to help! -->
-- [ ] The title for my package or theme uses its npm title (`hyper-plugin-example`)
-- [ ] The link for my package or theme uses the npmjs.com link (https://www.npmjs.com/package/hyper-plugin-example)
-- [ ] There is a visual representation of what my plugin or theme does in the repo. (Plugin: screenshot or gif || Theme: screenshot of the theme applied to Hyper)
-- [ ] Put your awesome item at the **BOTTOM** of the correct (plugin, theme, or resource) list.
-- [ ] **VERY IMPORTANT:** I've written a short (one sentence) description for my package or theme as the description of the awesome plugin, theme, or resource in the README.md file I'm submitting in the PR.
+- [ ] **Title:** The title for the addition uses its npm title (`hyper-example`) or the resource's title.
+- [ ] **Link:** The link for the addition uses the npmjs.com link (`https://www.npmjs.com/package/hyper-example`) or the resource's full URI.
```

Comment:
FWIW I was not aware it needed to be replaced.

I think there are multiple issues with the current format.
- We are asking contributors to add explicit title and description to their PRs. Then the package name is often already in the PR title, so it may feel **redundant** sometimes.
- The check boxes get a lot of attention: it feels like you just have to check them all to validate your PR, so people may just **rush** through it without reading everything.
- The use of "example" is misleading IMO. If it's really an example, I'm not supposed to change it. Since the format rules are already in `CONTRIBUTING.md`, this could be made more **explicit** like `<your package name here>` and `<your package URL here>`.